HelpSource/Guides/Internal-Snooping.schelp

```diff
@@ -57,38 +66,39 @@ Server.filenameSymbol.postln; // the path to the file that defined this class
 
 (
 // print all classes whose names start with 'F'
-Class.allClasses.do({ arg class;
-	if (class.name.asString.at(0) == $F, { class.name.postln; });
-})
+Class.allClasses.do { | class |
```

Comment:
I noticed you've changed this throughout the document. If you want to get started on a SC coding standards style guide, I would encourage it, but I think it's a little inconsiderate to start applying your syntax style preferences to code examples without a discussion beforehand. Also seems like it might not be the best use of time if we don't have standards/a rationale in place yet. FWIW, I personally find both versions of this code equally difficult to read, and I think `| |` is syntactic sugar to many new users.